CHANGELOG/v1.2.md

```diff
@@ -506,6 +506,11 @@ Other minor additions
   tail : ∀ {i} {j : Size< i} → Stream A i → Stream A j
   ```
 
+* Added new proofs to `Data.Bool`:
```

Comment:
Thanks for adding a CHANGELOG entry, most people forget that! The only problems is that `v1.2` has now been released so the entry should go in `CHANGELOG.md` outside of this folder which for old changelogs only.

Reply:
I moved to CHANGELOG.md.